### DISCUSSION TITLE
（Only China users）关于国区 App Store

### DISCUSSION THREAD

**Discussion by lollipopkit** [lollipopkit/flutter_server_box#180] [Announcements]
首先，国庆假期玩了几天，抱歉。

其次，因为现在需要备案才了在国区上架，所以可能需要一些时间才能在iOS上更新下个版本

![备案](https://github.com/lollipopkit/flutter_server_box/assets/10864310/eb0a5015-806f-4efa-8cc2-3905e8eeccc7)

~最后，一些废话，为了搞这个备案：~
- ~未能在备案官网(miit.gov.cn)找到备案入口（😄）~
- ~腾讯云要有剩余时长大于3个月的机器才能备案，专门花了几百在腾讯云买了服务器，这几百块对于一个没多少收入的独立开发者来说还是挺多的（感谢所有购买了的用户🙏）~

10.6更新:
备案被驳回了，理由是不支持个人备案上架应用商店（后来我改了备注：仅个人使用，还是不给过）

~因为上面的原因，所以打算转免费+内购的方式（内购价格依旧会保持低价；麻烦大家转去其他区再次获取App），目前所有功能都保持免费，但后继新增功能需要内购。~

~准备找个班上吧，这钱真不好挣。~

24.1.25更新:
上班了几个月了，不需要靠开源赚钱了，所以打算永久免费，我的其他App也是。但是如果各位投喂，来者不拒。😄

**Comment by ilworkcn** [comment]
备案政策对于个人开发者真是太难了，不过有一个问题想问一下 maintainer 您，如果后续有内购功能的话，之前在国区购买过的用户该做何种打算？

**Reply by lollipopkit** [reply]
难处是没法判断用户之前是否是付费购买的（🤷），所以只有一视同仁

新增功能不一定需要内购，内购功能肯定 不多 且 不贵，毕竟开源嘛（自从 ServerBox 之后，服务器管理 App 层出不穷，但我只是猜测，理性讨论）

个人比较想走捐赠的方式，但是感觉难，我这个年纪还可以写十来年吧，可以每种都试试 😄

**Reply by wc7086** [reply]
> 个人比较想走捐赠的方式

靠捐赠不现实，这种非生产力刚需的软件用户的付费转化率太低了，steamtool 一万五 star，用的富哥们儿也多一个月也就不到一万的发电。

KiCad 这种用户不少的生产力工具每年也就筹够开发资金。我的建议是老老实实找个班上，业余时间可以尝试开发一些更有想象力的，更有故事好拉投资的应用。

能赚到大钱的开源应用要么是做企业级解决方案得到认可，拉到不少投资。要么是得到灰产认可转闭源收到不少付费订阅。

**Reply by wc7086** [reply]
像 frp 的作者，七万多 star ，直接或间接用到的人最保守也有几百万了，除了金牌赞助商，一个月能不能收到 50 赞助都不一定。想靠开源赚到第一桶金还是得研究企业级解决方案，软件开源，方案收费。

或者做有想象力的能吸引到用户的 iOS 收费应用。安卓转化率太低环境也太卷除非做成大爆款，不然很难赚到比较多的钱。

**Reply by lollipopkit** [reply]
已经上班了（虽然成都、我这个技术栈的班不咋样）

企业级解决方案目前来看不太现实。

目前的想法是做一些中心化的服务来赚钱，可惜精力不足，只能期望能遇到志同道合的人（得在宣传上下力，这方面挺欠缺的）

**Comment by Java2Boy** [comment]
就这环境还支持创业，笑煞仙人哩

**Reply by lollipopkit** [reply]
～，或许就是为了让大家都去注册公司，会考虑（因为啥也不懂，出身太低）办个公司

**Comment by Twshuai** [comment]
国区用户的就无法更新了嘛

**Comment by AoEiuV020** [comment]
| 新增功能需要内购
那这些功能还会保持开源吗，

**Comment by AoEiuV020** [comment]
我刚还看见一个同类app备案通过了的，看他这意思挺顺利的，或许可以请教一下，
https://www.v2ex.com/t/983284

**Reply by lollipopkit** [reply]
我和他流程一样，但最后一步“不出意外-通过”我出意外了，不让过

**Comment by lollipopkit** [comment]
已完成备案
